src/index.ts

```diff
@@ -0,0 +1,347 @@
+    return {
+      patterns: {
+        opensea: NftTool.regexp,
+      }
+    };
+  }
+
+  /**
+   * Notify core that read-only mode is supported
+   */
+  static get isReadOnlySupported(): boolean {
+    return true;
+  }
+
+  /**
+   * Creates UI of a Block
+   */
+  render(): HTMLElement {
+    this.nodes.wrapper = document.createElement('div');
+    this.nodes.wrapper.classList.add(styles['nft-tool']);
+
+    /** Compose form */
+    this.nodes.form = document.createElement('div');
+    this.nodes.form.classList.add(styles['nft-tool__form']);
+
+    this.nodes.formNetworkWrapper = document.createElement('div');
+    this.nodes.formNetworkWrapper.classList.add(styles['nft-tool__form-param']);
+
+    this.nodes.formNetworkLabel = document.createElement('label');
+    this.nodes.formNetworkLabel.classList.add(styles['nft-tool__form-label']);
+    this.nodes.formNetworkLabel.innerHTML = 'Network';
+    this.nodes.formNetworkWrapper.appendChild(this.nodes.formNetworkLabel);
+    this.nodes.formNetworkSelect = document.createElement('select');
+    this.nodes.formNetworkSelect.classList.add(styles['nft-tool__form-select']);
+    this.nodes.formNetworkSelect.style.backgroundImage = `url(data:image/svg+xml;utf8,${encodeURI(
+      IconChevronDown
+    )})`;
+
+    [
+      { value: 'ethereum', label: 'Ethereum' },
+      { value: 'matic', label: 'Polygon' },
+    ].forEach((network) => {
+      const option = document.createElement('option');
+
+      option.value = network.value;
+      option.innerHTML = network.label;
+
+      if (this.data.network === network.value) {
+        option.selected = true;
+      }
+
+      this.nodes.formNetworkSelect.appendChild(option);
+    });
+    this.nodes.formNetworkWrapper.appendChild(this.nodes.formNetworkSelect);
+
+    this.nodes.formContractAddress = document.createElement('div');
+    this.nodes.formContractAddress.classList.add(styles['nft-tool__form-param']);
+    this.nodes.formContractAddressLabel = document.createElement('label');
+    this.nodes.formContractAddressLabel.classList.add(styles['nft-tool__form-label']);
+    this.nodes.formContractAddressLabel.innerHTML = 'Contract Address';
+    this.nodes.formContractAddressInput = document.createElement('input');
+    this.nodes.formContractAddressInput.classList.add(styles['nft-tool__form-input']);
+    this.nodes.formContractAddressInput.type = 'text';
+    this.nodes.formContractAddressInput.placeholder = '0x...';
+
+    this.nodes.formContractAddress.appendChild(this.nodes.formContractAddressLabel);
+    this.nodes.formContractAddress.appendChild(this.nodes.formContractAddressInput);
+
+    this.nodes.formTokenId = document.createElement('div');
+    this.nodes.formTokenId.classList.add(styles['nft-tool__form-param']);
+    this.nodes.formTokenIdLabel = document.createElement('label');
+    this.nodes.formTokenIdLabel.classList.add(styles['nft-tool__form-label']);
+    this.nodes.formTokenIdLabel.innerHTML = 'Token ID';
```

Comment:
please wrap all tokens with `this.api.i18n()` to allow users override them via i18n dictionary